package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oplog/resource-redux",
-  "version": "0.2.3",
+  "version": "0.2.5",
```

Comment:
Why not `0.2.4`?

Reply:
to be compatible with the npm package version